docker-storage-setup.sh

```diff
@@ -895,8 +883,8 @@ setup_storage() {
   # Set up lvm thin pool LV
   if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
     setup_lvm_thin_pool
-  elif [[ "$STORAGE_DRIVER" == "overlay" || "$STORAGE_DRIVER" == "overlay2" ]];then
-    setup_overlay
+  else
```

Comment:
I look at the devicemapper as the one off and everything else uses the generic code. At least that was what I was going for.  If we add more graphdrivers, then either they can use the default or we make a graphdriver specific patch.